pkg/volume/emptydir/empty_dir.go

```diff
@@ -227,7 +227,7 @@ func (ed *emptyDir) SetUpAt(dir string, mounterArgs volume.MounterArgs) error {
 		err = fmt.Errorf("unknown storage medium %q", ed.medium)
 	}
 
-	volume.SetVolumeOwnership(ed, mounterArgs.FsGroup)
+	volume.SetVolumeOwnership(ed, mounterArgs.FsGroup, nil)
```

Comment:
comment on the nil, please, for the sake of future readers :)

Reply:
fixed and fixed all other occurrences.